src/components/issue-blurb.jsx

```diff
@@ -10,144 +9,153 @@ import GithubFlavoredMarkdown from './gfm';
-        );
+    let icon = null;
+    if (isPullRequest) {
+      let state;
+      if (card.isPullRequestMerged()) {
+        state = 'merged';
       } else {
-        let iconType;
-        if (issue.state === 'open' && !issue.closedBy) {
-          iconType = IssueOpenedIcon;
-        } else if (issue.state === 'open' && issue.closedBy) {
-          iconType = IssueReopenedIcon;
-        } else if (issue.state === 'closed') {
-          iconType = IssueClosedIcon;
-        } else {
-          iconType = QuestionIcon;
-        }
-        icon = (
-          <iconType title='Click for Issue Details' className='blurb-icon' onClick={this.onClickIcon} data-state={issue.state}/>
-        );
+        state = issue.state;
       }
-
-      const classes = {
-        'issue-blurb': true,
-        'is-pull-request': isPullRequest,
-        'is-merged': card.isPullRequestMerged()
-      };
-
-      let kanbanColumnIcon;
-      const kanbanColumn = issue.labels.filter((label) => {
-        return getFilters().getState().columnRegExp.test(label.name);
-      })[0];
-      if (kanbanColumn) {
-        const {color, name} = kanbanColumn;
-        kanbanColumnIcon = (
-          <ColoredIcon color={color} name={name}>
-            <ListUnorderedIcon/>
-          </ColoredIcon>
-        );
-      }
-
-      return (
-        <span className={classnames(classes)}>
-          {kanbanColumnIcon}
-          <a className='blurb-number-link'
-            target='_blank'
-            href={issue.htmlUrl}
-            >
-            <span className='blurb-number'>{issue.number}</span>
-          </a>
-          <BS.OverlayTrigger
-            rootClose
-            trigger={['click', 'focus']}
-            placement='bottom'
-            overlay={bodyPopover}>
-            <span className='-just-for-overlay-trigger'>
-              {icon}
-              {privateEl}
-              <span className='blurb-secondary-repo'>{multipleRepoName}</span>
-              {blurbContext}
-            </span>
-          </BS.OverlayTrigger>
-        </span>
+      icon = (
+        <GitPullRequestIcon
+          title='Click for Pull Request Details'
+          className='blurb-icon'
+          onClick={onClickIcon}
+          data-state={state}/>
       );
     } else {
-      // no Issue found
-      return (
-        <span className='issue-blurb'>
-          <span className='blurb-number'>{card.number}</span>
-          {privateEl}
-          <span className='blurb-secondary-repo'>{multipleRepoName}</span>
-          {blurbContext}
-        </span>
+      let IconType;
+      if (issue.state === 'open' && !issue.closedBy) {
+        IconType = IssueOpenedIcon;
+      } else if (issue.state === 'open' && issue.closedBy) {
+        IconType = IssueReopenedIcon;
+      } else if (issue.state === 'closed') {
+        IconType = IssueClosedIcon;
+      } else {
+        IconType = QuestionIcon;
+      }
+      icon = (
+        <IconType
```

Comment:
unnecessary style change

Reply:
iconType is renamed to IconType. Should that be considered as syntax change?

Reply:
given that variable above needed to be renamed, renaming this is appropriate even if not required, and it is a change to the code, so style changes are ok.